aiosmtpd/smtp.py

```diff
@@ -928,9 +936,10 @@ async def smtp_RCPT(self, arg: str) -> None:
             await self.push(syntaxerr)
             return
         address, params = self._getaddr(arg)
+        if address is None:
+            return await self.push("553 5.1.3 Error: malformed address")
```

Comment:
Since you're using this string twice, maybe refactor it into a constant?

Reply:
In essence, I agree. But I do have the idea of making _all_ SMTP status code string into a class of messages, similar (but not identical) to [this file](https://github.com/pepoluan/aiosmtpd/blob/pytest_based/aiosmtpd/testing/statuscodes.py) (part of PR #202 ).

Part of the reason is to make it easier to change if necessary (no need to hunt through all `smtp.py`), and some thoughts about i18n (though I honestly can't find info if SMTP status messages are allowed in non-English...). Maybe by making it into some sort of "pluggable" class, similar to handler?